### ISSUE TITLE
UESolDS dataset

### ISSUE THREAD

**Issue by zchwang** [Violet969/PLM_Sol#1]
Hi, 

Nice work. The number of training samples in the embedding_dataset directory seems to be fewer than reported in the paper (32,053 Insol vs. 47,291 Sol). Is the full dataset available? Additionally, what do the two labels A-1 and A-0 refer to?

Best regards,

**Comment by Violet969** [comment]
Hi, the UESolDS dataset includes a training set, validation set, and test set, so the size of the training set needs to be reduced to account for the validation and test sets. Additionally, the two labels are A-1 for soluble proteins and A-0 for insoluble proteins.
